atomic_reactor/plugins/inject_yum_repos.py

```diff
@@ -260,14 +267,14 @@ def run(self):
                 fetched_yum_repos[repourl] = yum_repo
 
         if not self.yum_repos:
```

Comment:
Do we need to keep this check here now even if we are returning in _inject_repo_files if there are no repos for the platform?

Reply:
If we want to keep the `no repos for <platform>` message a warning then yes. Not having _any_ repos is normal

But I can remove the check and reduce the log level to info

Reply:
I don't mind then, you can keep it as it is.

Reply:
this makes sens, this is for when there aren't ANY repos at all, the warning is when there aren't any repos for specific arch